### PR TITLE
RAT-394: fix to make all code use DefaultLog (Harmonize - Part 4)

### PR DESCRIPTION
**Pull request by Claudenw**
Part of RAT-323

Remove most passing of LOG instances in favour of using DefaultLog.getInstance() to retrieve it where necessary.

### CODE REVIEW

File: apache-rat-core/src/main/java/org/apache/rat/commandline/Arg.java [hunk 1]
```diff
@@ -659,4 +675,66 @@ public static void reset() {
...
+        }
+        return null;
+    }
+
+    /**
+     * Finds the Arg that contains an Option with the specified key.
+     * @param key the key for the Option to locate.
+     * @return The Arg or {@code null} if no Arg is found.
+     */
+    public static Arg findArg(String key) {
```
Comment (ottlinger) [inline apache-rat-core/src/main/java/org/apache/rat/commandline/Arg.java:700]: Is NPE-safeness an issue here?

File: apache-rat-core/src/main/java/org/apache/rat/commandline/Arg.java [hunk 2]
```diff
@@ -659,4 +675,66 @@ public static void reset() {
             }
         }
     }
+
+    /**
+     * Finds the Arg that an Option is in.
+     * @param optionToFind the Option to locate.
+     * @return The Arg or {@code null} if no Arg is found.
+     */
+    public static Arg findArg(Option optionToFind) {
```
Comment (ottlinger) [inline apache-rat-core/src/main/java/org/apache/rat/commandline/Arg.java:684]: Is NPE-safeness an issue here?

**Comment by ottlinger** [comment]
@Claudenw except from the 2 NPE-questions everything looks fine. I had the impression that we introduced the Log and are now removing it again .....

File: apache-rat-core/src/main/java/org/apache/rat/commandline/Arg.java (hunk 1 above)
Comment (Claudenw) [reply apache-rat-core/src/main/java/org/apache/rat/commandline/Arg.java:702]: Yes. fixed

File: apache-rat-core/src/main/java/org/apache/rat/commandline/Arg.java (hunk 2 above)
Comment (Claudenw) [reply apache-rat-core/src/main/java/org/apache/rat/commandline/Arg.java:684]: Yes. fixed

**Comment by ottlinger** [comment]
Thanks for all your work - pls refresh the other pull requests to fetch changes from master.

**Pull request merged**